tools/integrations/manage.sh

```diff
@@ -69,6 +69,12 @@ cmd_publish() {
   echo "Publishing new version ($versioned_image)"
   docker push "$versioned_image"
   docker push "$latest_image"
+
+  # publish spec to cache. do so, by running get spec locally and then pushing it to gcs.
+  local tmp_spec_file; tmp_spec_file=$(mktemp)
+  docker run --rm "$versioned_image" spec | jq .spec > "$tmp_spec_file"
+  gcloud auth activate-service-account --key-file /Users/charles/Downloads/prod-ab-cloud-proj-bdb658ebbe5a.json
```

Comment:
so this line is obviously wrong (since it mentions the FS on my local machine). As far as I can tell this script gets called from 2 places: 1. from people's local machines 2. from GH actions. I think my inclination is to add another argument for `cmd_publish` where the user has to publish the path to the service account key. @sherifnada does this feel like a reasonable way of going about it? It's a credential so it has to add some developer friction, but I'm trying to keep it as light as possible. 

I guess it should be designed so that if someone doesn't pass the cred but their `gsutil` is already authed into a user that can access the bucket then it should still work as well.

Reply:
seems reasonable to me to limit publishing new connector versions to only Github

Reply:
kk. well it's possible in both places now :D